facia/app/services/dotcomrendering/FaciaPicker.scala

```diff
@@ -172,10 +96,10 @@ object FaciaPicker extends GuLogging {
     if (isRss) LocalRender
     else if (forceDCROff) LocalRender
     else if (forceDCR) RemoteRender
-    else if (isNetworkFront)
-      if (dcrCouldRender && isInNetworkFrontTest) RemoteRender else LocalRender
-    else if (dcrCouldRender && dcrSwitchEnabled) RemoteRender
-    else LocalRender
+    else if (dcrCouldRender && dcrSwitchEnabled) {
+      if (isNetworkFront) if (isInNetworkFrontTest) RemoteRender else LocalRender
+      RemoteRender
```

Comment:
```suggestion
     else RemoteRender
```

You need an else here or else this block of code is always going to return RemoteRender

https://scastie.scala-lang.org/o8hV5G5OQzyGyZy14irtDg

Reply:
Absolutely, thanks @AshCorr!